test/helper.js

```diff
@@ -68,22 +52,20 @@ helper.setupApi = function() {
     authServer = await base.testing.createMockAuthServer({
       port: 60407,
       clients:  defaultClients,
-      credentials: cfg.get('taskcluster:credentials')
+      credentials: cfg.taskcluster.credentials
     });
 
     // Create Hooks table
-    helper.Hook = data.Hook.setup({
-      account:      cfg.get('hooks:azureAccount'),
-      table:        cfg.get('hooks:hookTableName'),
-      credentials:  cfg.get('taskcluster:credentials'),
-      authBaseUrl:  cfg.get('taskcluster:authBaseUrl'),
-      signingKey:   cfg.get('hooks:tableSigningKey'),
-      cryptoKey:    cfg.get('hooks:tableCryptoKey'),
-      process:      'testing'
-    });
+    helper.Hook = await load('Hook', loadOptions);
+
+    // Remove all entities before each test
+    await helper.Hook.scan({}, {handler: hook => hook.remove()});
 
     helper.creator = new taskcreator.MockTaskCreator()
-    webServer = await bin.server(testProfile, {taskcreator: helper.creator});
+    webServer = await load('server', _.defaults({
+      Hook: helper.Hook,
+      taskcreator: helper.creator,
```

Comment:
Yes, this is pretty neat.  The application structure was already pretty nicely set up for dependency injection, but the loader makes it *much* easier.